app/ai/voice/agents/breeze_buddy/memory/worker.py

```
@@ -0,0 +1,250 @@
+    if phone and explicit_customer_id and key_type == "phone":
+        try:
+            await _upsert_alias_and_merge(
+                reseller_id=reseller_id,
+                merchant_id=merchant_id,
+                phone=phone,
+                customer_id=explicit_customer_id,
+                old_phone_key=customer_key,
+            )
+            customer_key = explicit_customer_id
+            key_type = "customer_id"
+        except Exception as merge_err:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)